File: The Mind/ClientApp/src/app/models/class/card.ts
export class Card {
  constructor(
    public value: number, // The value of the card, from 1 to 100
    public isFaceUp: boolean,
    public playerIndex : number = 0,
    public rotation : number = 0,
    public xOffset : number = 0,
    public yOffset : number = 0,
    public isPlayable : boolean = true,
  ) {}
}

//dummy data
export const CARDS: Card[] = [
  new Card(1, true),
  new Card(2, true),
  new Card(3, true),
  new Card(4, true),
  new Card(5, true),
]

File: The Mind/Models/Classes/Card.cs
namespace The_Mind.Models;

public class Card
{
    public Card(int value, bool isFaceUp = true, bool isPlayable = true)
    {
        Value = value;
        this.isFaceUp = isFaceUp;
        this.isPlayable = isPlayable;
    }

    public int Value { get; set; }
    public bool isFaceUp { get; set; }
    public bool isPlayable { get; set; }
}